framework/test/gtest/FrameworkFactoryTest.cpp

```diff
@@ -27,7 +27,7 @@ limitations under the License.
 
 using namespace cppmicroservices;
 
-TEST(FrameworkFactoryTest, FrameworkInstantiation)
+TEST(FrameworkFactoryTest, testFrameworkInstantiation)
```

Comment:
Just curious... why change the name?

Reply:
We have test points starting with 'Test', 'test' or just the name.
I am thinking, may be, we can start all the test-point names with 'test' and make it a standard.